### PR TITLE
Feat/user edit

### PR DESCRIPTION
**Pull request by jnizet**
on top of #35

### CODE REVIEW

File: frontend/src/app/user-edit/user-edit.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    const activatedRoute = {
+      snapshot: {data: {}}
+    };
+
+    beforeEach(() => TestBed.configureTestingModule({
+      imports: [AppModule, RouterTestingModule],
+      providers: [{provide: ActivatedRoute, useValue: activatedRoute}]
+    }));
+
+    it('should expose no created user, and a default user', async(() => {
```
Comment (cexbrayat) [inline frontend/src/app/user-edit/user-edit.component.spec.ts]: This `async` is probably unnecessary.

File: frontend/src/app/user-edit/user-edit.component.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  save() {
+    if (this.editedUser) {
+      this.userService.update(this.editedUser.id, this.user)
+        .subscribe(() => this.router.navigate(['/users']));
+    }
+    else {
+      this.userService.create(this.user)
+        .subscribe(createdUser => this.createdUser = createdUser);
```
Comment (cexbrayat) [inline frontend/src/app/user-edit/user-edit.component.ts]: No error handling for create/update?

File: frontend/src/app/users/users.component.spec.ts [hunk 3]
```diff
@@ -1,4 +1,4 @@
-import { TestBed } from '@angular/core/testing';
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
```
Comment (cexbrayat) [inline frontend/src/app/users/users.component.spec.ts]: new imports are probably useless

File: frontend/src/app/user-edit/user-edit.component.ts (hunk 2 above)
Comment (jnizet) [inline frontend/src/app/user-edit/user-edit.component.ts]: I added an issue. We need to choose how to handle such errors.

**Pull request merged**